norduser/service/fork.go

```diff
@@ -170,6 +171,16 @@ func (c *ChildProcessNorduser) Stop(uid uint32) error {
 		return fmt.Errorf("sending SIGTERM to norduser process: %w", err)
 	}
 
+	if wait {
+		proc, err := os.FindProcess(pid)
+		if err == nil {
+			_, err := proc.Wait()
```

Comment:
Is `Wait` needed? Daemon is already waiting for the process(line 143).

Reply:
It is waiting in a goroutine, so the `Stop` function returns immediately. The purpose of `Wait` here is to block `Stop` the return of function until the norduserd process actually exits.